bin/bash_stub.sh

```diff
@@ -64,17 +64,21 @@ function extract-string-properties {
 }
 
 function print-output {
-  case ${1} in
+  local type=${1}
+  local target=${2}
```

Comment:
what exactly is target here?

Reply:
In this case target is the file name (or dummy values for stdout and stderr, who don't use it)